src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
Consider following the encapsulation principle more strictly. The `fruitStorage` map should be private to prevent direct access and modifications from outside the class. This will help in maintaining the integrity of the storage data.